arkiverer/src/test/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/service/fileservice/FilestorageServiceTests.kt

```diff
@@ -119,13 +119,9 @@ class FilestorageServiceTests {
 		verifyMockedGetRequests(1, makeUrl(fileIdsAndResponses.drop(filesInOneRequestToFilestorage * 2).take(1)))
 	}
 
-	@Disabled
 	@Test
-	fun `getFilesFromFilestorage - Asking for 3 files - Only 2 is returned - will throw exception`() {
```

Comment:
Old test case no longer valid - Soknadsfillager cannot respond with a different amount of files than what was asked for. Updating with new test that captures new behaviour.